AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/utils.py

```diff
@@ -13,7 +13,10 @@
 from typing import Callable, Tuple
 
 import azlmbr
-import azlmbr.legacy.general as general
+try:
+    import azlmbr.legacy.general as general  # Editor test.
+except ModuleNotFoundError:  # MaterialEditor test.
+    import azlmbr.atomtools.general as general
```

Comment:
asking dev about this

Reply:
after reviewing this file, it seems the only shared functions are the ones that call `idle_wait_frames` or `test_output` so I could create a new file/module specifically for MaterialEditor that will be a copypasta of some of the same functions but it will utilize the `azlmbr.atomtools.general as general` import and then the `utils.py` here will remain in use for Editor with its existing `azlmbr.legacy.general as general` import

or we can leave this try-catch as is...let me know what you think is better and I am fine with either option

Reply:
after consulting with @gadams3 a GHI has been cut to deal with this: https://github.com/o3de/o3de/issues/11056 
the try-except block will be updated as well to import `azlmbr.atomtools.general` first then import `azlmbr.legacy.general` if the import fails